.github/workflows/test.yaml

```diff
@@ -2,6 +2,10 @@ on: ["push", "pull_request"]
 
 name: Test
 
+concurrency:
+  group: ${{ github.head_ref }}
```

Comment:
Vyper is using `${{ github.workflow }}-{{ github.head_ref }}` here:

https://github.com/vyperlang/vyper/pull/3179